lib/cocoapods/installer/target_installer/pod_target_installer.rb

```diff
@@ -165,6 +166,18 @@ def create_xcconfig_file
         end
       end
 
+      # Links a module map and an umbrella header to Public headers.
+      #
+      # @return [void]
+      #
+      def link_module_map
```

Comment:
This method is doing something else than it's name let assume, I would split that either in two methods or just do it in the methods responsible for creating the files.